src/qt/toolbar.cpp

```diff
@@ -58,7 +58,7 @@ class wxQtToolButton : public QToolButton, public wxQtSignalHandler
 
 public:
     wxQtToolButton(wxToolBar *handler, wxToolBarTool *tool)
-        : QToolButton(handler->GetHandle()),
+        : QToolButton(handler != nullptr ? handler->GetHandle() : nullptr),
```

Comment:
Is it really normal for `handler` to be null here? When does it happen? A comment explaining this would be nice if it really must be supported (otherwise, preventing it from happening would be even nicer).

Reply:
Searching through the code, it seems like this change and wxQtActionGroup one is not necessary, I will remove them.